README.md

```diff
@@ -37,7 +37,7 @@ Start a cluster:
 
 Add more brokers:
 
-- ```docker-compose scale kafka=3```
+- ```docker-compose up -d --scale kafka=3```
```

Comment:
No issue with this method, it's just that it's deprecated and at some point it will not work anymore.

I understood that you wanted to show how to dynamically resize the cluster and it will be also the case is you're using the `up` command.

If you're running first `docker-compose up -d` then `docker-compose up -d  --scale kafka=3`, you'll see that it will dynamically resize the cluster (the first kafka instance will not be recreated).

```
e27993b28e8b        wurstmeister/kafka:0.11.0.1   "start-kafka.sh"         *Less than a second ago*   Up 2 seconds        0.0.0.0:9094->9092/tcp                               *kafka_kafka_3*
8eff61bb8802        wurstmeister/kafka:0.11.0.1   "start-kafka.sh"         *Less than a second ago*   Up 2 seconds        0.0.0.0:9093->9092/tcp                               *kafka_kafka_2*
12f5ced5d20a        wurstmeister/zookeeper        "/bin/sh -c '/usr/sb…"   About a minute ago       Up 13 seconds       22/tcp, 2888/tcp, 3888/tcp, 0.0.0.0:2181->2181/tcp   kafka_zookeeper_1
f7961bd565a3        wurstmeister/kafka:0.11.0.1   "start-kafka.sh"         *About a minute ago*       Up 12 seconds       0.0.0.0:9092->9092/tcp                               *kafka_kafka_1*
```

But agree with you the previous method to scale was more explicit.